var/spack/repos/builtin/packages/py-dace/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyDace(PythonPackage):
+    """DaCe is a fast parallel programming framework that takes code in
+    Python/NumPy and other programming languages, and maps it to
+    high-performance CPU, GPU, and FPGA programs, which can be
+    optimized programmatically or interactively."""
+
+    homepage = "https://github.com/spcl/dace"
+    pypi = "dace/dace-0.15.1.tar.gz"
+    git = "https://github.com/spcl/dace.git"
+
+    maintainers("tbennun")
+
+    version("master", branch="master", submodules=True)
+    version("0.15.1", sha256="69bfdbbd5c7177f2926a874f5fa82fcdef61fc532c022b4bc12e1e9218724093")
+
+    variant(
```

Comment:
Are there no differences in the build for this variant?

Reply:
at the very end a dependency on Likwid is added. The package detects if it is installed at runtime.

Reply:
Yeah I saw the dependendency but I was worried about the lack of a build option.

Reply:
Welcome to Python 😆